pkg/operator/ceph/object/config.go

```diff
@@ -251,6 +266,28 @@ func (c *clusterConfig) generateMonConfigOptions(rgwConfig *rgwConfig) (map[stri
 	return configOptions, nil
 }
 
+func (c *clusterConfig) setObectStoreReferenceAnnotation(secret *v1.Secret) error {
```

Comment:
I think it would be good to have a unit test for this. The logic looks good to me, but double checking, and future regression testing will be good

Reply:
done